stonesoup/mixturereducer/gaussianmixture.py

```diff
@@ -161,8 +175,7 @@ def merge(self, components_list):
             # (modifying list in loop, so copy used)
             for component in remaining_components.copy():
                 # Calculate distance between component and best component
-                distance = dist.mahalanobis(
-                    best_component.mean[:, 0], component.mean[:, 0], best_component.covar)
+                distance = (Mahalanobis(state1=component, state2=best_component))**2
```

Comment:
```suggestion
                distance = (Mahalanobis()(state1=component, state2=best_component))**2
```
May also just want to define `Mahalanobis()` once to avoid recreating every loop.